README.md

```diff
@@ -1 +1,75 @@
-# Hello-Microverse
\ No newline at end of file
+📖 Hello Microverse
+
+Hello Microverse is a simple project. It is the first project created as part of microverse curriculam.
+
+🛠 Built With
+Tech Stack
+
+Client HTML/CSS
+Server NA
+Database NA
+Key Features
+Describe between 1-3 key features of the application.
+
+It display "Hello Microverse!" when run in a browser
+
+💻 Getting Started
+
+To get a local copy up and running, follow these steps.
+
+Prerequisites
+In order to run this project you need:
+A Code editor such as VSCode or atom and a browser
+
+Setup
+Clone this repository to your desired folder and double click on index.html file.
+
```

Comment:
- [ ] As request by the previous code reviewer , you should use [this template](https://github.com/microverseinc/readme-template) provided by microverse to document your project, it will make it more professional 👍 
- [ ] Take a look at it and customized it with data related to your project by filling every section find in the README template